zabbix_metrics_watcher.py

```diff
@@ -214,14 +217,33 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                now = datetime.datetime.now()
+                if build_name not in builds_in_new:
+                    builds_in_new[build_name] = now
+
+            elif status != 'New':
+                try:
+                    del builds_in_new[build_name]
```

Comment:
No, it can be a running build, which was not stored in `builds_in_new`.

I guess the part with 'check that builds_in_new is empty' can be moved below to `spent_in_new` calculation